common/src/test/resources/logback.xml

```diff
@@ -16,17 +16,22 @@
     <ReconnectionDelay>10000</ReconnectionDelay>
     <IncludeCallerData>true</IncludeCallerData>
     <RemoteHosts>${LILITH_HOSTNAME:-localhost}</RemoteHosts>
+    <!-- Disables Lilith ClassicMultiplexSocketAppender since its use causes hanging
```

Comment:
I agree with you that it does not disable Lilith. The connection is established with HiveServer2 instead of Lilith server and unit tests hang when `MultiplexSendBytesService` calls `sendBytes()` method. 
I have updated changes, so now Lilith is disabled.